CHANGELOG.md

```diff
@@ -2,208 +2,234 @@
 
 ## Upcoming version
 
-## [v0.15.0]
+## \[v0.16.0\]
 
 ### Added
-- [[#270](https://github.com/rust-vmm/vm-memory/pull/270)] atomic_bitmap: add capability to reset bits range
-- [[#285](https://github.com/rust-vmm/vm-memory/pull/285)] Annotated modules in lib.rs to indicate their feature
+
+- \[[#287](https://github.com/rust-vmm/vm-memory/pull/287)\] Add build support on riscv64.
```

Comment:
Mh, we have a bit more than just build support, right? E.g. we run CI and everything, so I think we can just say we added "support for riscv64". We also need to update the list in the readme, now that I'm thinking about it!

Reply:
Noted